js/index.js

```diff
@@ -0,0 +1,16 @@
+const today = new Date();
+const thisYear = today.getFullYear();
+const footer = document.querySelector('footer');
+const copyright = document.createElement('p');
+copyright.innerHTML = `Dewi Anggraini ${thisYear}`;
+footer.appendChild(copyright);
+const skills = ['JavaScript', 'HTML', 'CSS', 'Problem solving'];
+const skillsSection = document.getElementById('skills');
+const skillsList = skillsSection.querySelector('ul');
+for (let i of skills){
```

Comment:
Better to use a more descriptive name like 'skillItem'.  Each skill in the list will be assigned one at a time to the let variable.